src/main/java/com/elbraulio/ezload/sql/InsertFromParser.java

```diff
@@ -103,13 +103,21 @@ public InsertFromParser(Parser parser, BuildSql buildSql, EzLogger logger) {
     @Override
     public int[] execute(Connection connection, BufferedReader bufferedReader)
             throws EzException {
+        /*
+         * @todo this method from InsertFromParser is too complex
+         * @body # from #40
```

Comment:
```suggestion
         * @body > from #40
```
change header by citation.